packages/webapp/src/app/page.tsx

```diff
@@ -1,113 +1,60 @@
-            </span>
-          </h2>
-          <p className="m-0 max-w-[30ch] text-sm opacity-50">
-            Explore starter templates for Next.js.
-          </p>
-        </a>
 
-        <a
-          href="https://vercel.com/new?utm_source=create-next-app&utm_medium=appdir-template&utm_campaign=create-next-app"
-          className="group rounded-lg border border-transparent px-5 py-4 transition-colors hover:border-gray-300 hover:bg-gray-100 hover:dark:border-neutral-700 hover:dark:bg-neutral-800/30"
-          target="_blank"
-          rel="noopener noreferrer"
-        >
-          <h2 className="mb-3 text-2xl font-semibold">
-            Deploy{" "}
-            <span className="inline-block transition-transform group-hover:translate-x-1 motion-reduce:transform-none">
-              -&gt;
-            </span>
-          </h2>
-          <p className="m-0 max-w-[30ch] text-balance text-sm opacity-50">
-            Instantly deploy your Next.js site to a shareable URL with Vercel.
-          </p>
-        </a>
-      </div>
-    </main>
-  );
-}
+  export default function LoginPage() {
+    const captchaRef = useRef<any>(null);
+
+    const router = useRouter();
+  
+
+  
+      const handleSubmit = (e: FormEvent<HTMLFormElement>) => {
+          e.preventDefault();
+          const captchaValue = captchaRef.current.getValue()
+        if (!captchaValue) {
+      //if catcha value is null, send a alert
+      alert('Please verify the reCAPTCHA!')}
+      else{
+      // Add your login logic here
+          router.push('/home')}
+          
+
+      };
+  
+      return (//Login form
+          <main className="bg-[#26313c] h-screen flex items-center justify-center p-10">
+            <div className="grid w-full h-full grid-cols-1 bg-white md:grid-cols-2">
+                <div className="bg-[#16202a] text-white flex items-center justify-center flex-col">
+                <h1 className="text-3xl mb-4 font-semibold ">AutoGrader</h1>
+                <form className="w-1/2 " onSubmit={(e) => {handleSubmit(e)}}>
+            
+                  <Label htmlFor="email">Email</Label>
+                  <Input className=" w-full mt-3 mb-4 bg-transparent rounded-full" type="email" id="email" />
+                  <Label htmlFor="email">  Password</Label>
+                  <Input className=" w-full mt-3 mb-4 bg-transparent rounded-full" type="password" id="password" />
+                   <ReCAPTCHA  className=" w-full mt-3 mb-4 flex justify-center" sitekey="6Lfj0M4pAAAAADIXRkl_0fUWhvXVnzeygsi1WFLr" ref={captchaRef} /> 
+                  <Button className="w-full mt-4 bg-indigo-600 rounded-full hover:bg-indigo-700" type="submit">Login</Button>
+               </form>
+               </div>
+               <div className="relative hidden md:block">
+                  <Image className="object-cover " fill={true} src="/bg.jpg" alt="bg-image"/>
+               </div>
+            </div>
+            
+        </main>
+      );
+     
+  }
+
```

Comment:
Good job on building the first screen of the project. 

How are you planning to store logged in user information on the front end ? So that when logged in user refreshes the application, they remain logged in and not have to log in again.

Here are some changes that I would like to see before merging this to main: 

- [ ] Extract login to a separate component
- [ ] Implement session management to store logged in user's details
- [ ] Redirect to login component if user not logged in 
- [ ] Redirect to landing page if user is logged in

Reference : 
https://nextjs.org/docs/app/building-your-application/routing/linking-and-navigating